docs/timeseries/analysis.rst

```diff
@@ -0,0 +1,213 @@
+.. _timeseries-analysis:
+
+Manipulation and analysis of time series
+****************************************
+
+.. |Table| replace:: :class:`~astropy.table.Table`
+.. |TimeSeries| replace:: :class:`~astropy.timeseries.TimeSeries`
+.. |BinnedTimeSeries| replace:: :class:`~astropy.timeseries.BinnedTimeSeries`
+
+Combining time series
+=====================
+
+The  :func:`~astropy.table.vstack`, and :func:`~astropy.table.hstack` functions
+from the :mod:`astropy.table` module can be used to stack time series in
+different ways.
+
+Time series can be stacked 'vertically' or row-wise using the
+:func:`~astropy.table.vstack` function (although note that sampled time
+series cannot be combined with binned time series and vice-versa)::
+
+    >>> from astropy.table import vstack
+    >>> from astropy import units as u
+    >>> from astropy.timeseries import TimeSeries
+    >>> ts_a = TimeSeries(time='2016-03-22T12:30:31',
+    ...                          time_delta=3 * u.s,
+    ...                          data={'flux': [1, 4, 5, 3, 2] * u.mJy})
+    >>> ts_b = TimeSeries(time='2016-03-22T12:50:31',
+    ...                          time_delta=3 * u.s,
+    ...                          data={'flux': [4, 3, 1, 2, 3] * u.mJy})
+    >>> ts_ab = vstack([ts_a, ts_b])
+    >>> ts_ab
+    <TimeSeries length=10>
+              time            flux
+                              mJy
+             object         float64
+    ----------------------- -------
+    2016-03-22T12:30:31.000     1.0
+    2016-03-22T12:30:34.000     4.0
+    2016-03-22T12:30:37.000     5.0
+    2016-03-22T12:30:40.000     3.0
+    2016-03-22T12:30:43.000     2.0
+    2016-03-22T12:50:31.000     4.0
+    2016-03-22T12:50:34.000     3.0
+    2016-03-22T12:50:37.000     1.0
+    2016-03-22T12:50:40.000     2.0
+    2016-03-22T12:50:43.000     3.0
+
+Time series can also be combined 'horizontally' or column-wise with other tables
+using the :func:`~astropy.table.hstack` function, though these should not be
+time series (as having multiple time columns would be confusing)::
```

Comment:
I wonder if an example should be given about combining time series? I.e., show how you'd do this but drop the second time column or something.  I'm certain there are use cases where someone would want to combine/match time series' after all.  E.g., the `simple_downsample` applied to a more finely sampled time series and then re-combining it with a less-sampled one.

Alternatively this could be punted for now and put on the feature list as "tool to combine time series" that would check for equality an the like...

Reply:
I think what you are asking for is basically the ability to use ``join`` with time series objects since that does the kind of combination you are talking about - except that ``join`` doesn't work with time columns. This might not be trivial, so I'm opening an issue here: https://github.com/astropy/astropy/issues/8586